tests/upgrade_script/static/falco_disabled.log

```diff
@@ -0,0 +1,8 @@
+
+[INFO]    falco will be disabled. Change "falco.enabled" to "true" if you want to enable it
+
+[INFO]    Replacing tail-db/tail-containers-state.db to tail-db/tail-containers-state-sumo.db
```

Comment:
Hm, that doesn't seem right 🤔 Why we do something with fluentbit when only falco has been changed?